flutter_local_notifications/example/lib/main.dart

```diff
@@ -183,12 +182,11 @@ class _HomePageState extends State<HomePage> {
   @override
   void initState() {
     super.initState();
-    _requestPermissions();
     _configureDidReceiveLocalNotificationSubject();
     _configureSelectNotificationSubject();
   }
 
-  void _requestPermissions() {
+  void _requestIOSPermissions() {
```

Comment:
Based on what I mentioned above, it this can be invoked on startup then the old name can be kept. Note that the name of `_requestIOSPermissions()` is incorrect as it also handles requesting permissions on macOS